pkg/resourcegenerator/prometheus/prometheus.go

```diff
@@ -0,0 +1,39 @@
+package prometheus
+
+import (
+	"fmt"
+
+	"github.com/kartverket/skiperator/api/v1alpha1"
+	"github.com/kartverket/skiperator/pkg/reconciliation"
+	"github.com/kartverket/skiperator/pkg/resourcegenerator/resourceutils/generator"
+	"github.com/kartverket/skiperator/pkg/util"
+	pov1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
+	prommodel "github.com/prometheus/common/model"
+)
+
+var (
+	multiGenerator        = generator.NewMulti()
+	defaultScrapeInterval = pov1.Duration("60s")
+	minimumInterval, _    = prommodel.ParseDuration("15s")
```

Comment:
Any reason for 15s as minimum? 10s could be an ok minimum as well 🤔

Reply:
I want to err on the side of caution regarding this change. Let's adjust this if there is relevant user feedback.